src/dotnet/APIView/ClientSPA/src/app/_helpers/common-helpers.ts

```diff
@@ -0,0 +1,29 @@
+export const REVIEW_ID_ROUTE_PARAM = "reviewId";
+export const ACTIVE_API_REVISION_ID_QUERY_PARAM = "activeApiRevisionId";
+export const DIFF_API_REVISION_ID_QUERY_PARAM = "diffApiRevisionId";
+export const DIFF_STYLE_QUERY_PARAM = "diffStyle";
+export const SCROLL_TO_NODE_QUERY_PARAM = "nId";
+
+export function getLanguageCssSafeName(language: string): string {
+  switch (language.toLowerCase()) {
+    case "c#":
+      return "csharp";
+    case "c++":
+      return "cplusplus";
+    default:
+      return language.toLowerCase();
+  }   
+}
+
+export function mapLanguageAliases(languages: Iterable<string>): string[] {
+  const result: Set<string> = new Set<string>();
+  
+  for (const language of languages) {
+    if (language === "TypeSpec" || language === "Cadl") {
```

Comment:
I don't think we need to maintain Cadl anymore. We should find out any reviews with Cadl as language in cosmos and mark them as deleted and also remove the reference to it. We can do this task to remove Cadl from code and DB as a separate task though.